akd/src/directory.rs

```diff
@@ -681,10 +681,12 @@ mod tests {
     async fn test_empty_tree_root_hash() -> Result<(), AkdError> {
         let db = AsyncInMemoryDatabase::new();
         let vrf = HardCodedAkdVRF {};
-        let akd = Directory::<_, _>::new::<Blake3>(&db, &vrf, false).await?;
+        let akd = Directory::<_, _>::new::<Blake3_256<BaseElement>>(&db, &vrf, false).await?;
 
         let current_azks = akd.retrieve_current_azks().await?;
-        let hash = akd.get_root_hash::<Blake3>(&current_azks).await?;
+        let hash = akd
+            .get_root_hash::<Blake3_256<BaseElement>>(&current_azks)
```

Comment:
nit: Here too, `Blake3` is just cleaner as a type alias

Reply:
Same response as above for this test.